src/Genumerics/DefaultNumericOperations.cs

```diff
@@ -1597,6 +1935,12 @@ public decimal DivRem(decimal dividend, decimal divisor, out decimal remainder)
 
         /// <inheritdoc />
         public BigInteger ToBigInteger(BigInteger value) => value;
+
+        /// <inheritdoc />
+        public BigInteger ToBigInteger(nint value) => new (value);
```

Comment:
Does this not bind to the `int` overload which then in a 64-bit context if the value is beyond `int.MinValue` or `int.MaxValue` the value is truncated. I think you need to check `IntPtr.Size`.

Reply:
No. There's no implicit `nint` to `int` conversion. This is equivalent to `new BigInteger((long)value)`

Reply:
Perfect, thank you!